lib/snippets/deploy-to-gke.rb

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env ruby
+
+# Usage: deploy-to-gke <app's namespace> <gcloud deployment> <gcloud project>
+
+# Requires the following to be available on Shipit machine:
+#  - gcloud binary available on the shipit machine's path
+#  - kubectl binary available in the shipit machine's path
+#  - GCLOUD_CREDENTIALS_DIR: location of JSON service account auth files, e.g. /home/username/.google_iam
```

Comment:
I think this is something you got from my gist, but the example is weird. :P
How about: "i.e. `$GCLOUD_CREDENTIALS_DIR/my-gcloud-project-name.json` must exist"